ci-operator/step-registry/openshift-hyperfleet/e2e/test/openshift-hyperfleet-e2e-test-commands.sh

```
@@ -39,4 +56,4 @@ export API_CHART_REF="${API_CHART_REF:-main}"
 export API_CHART_PATH="${API_CHART_PATH:-charts}"
 
 # Run e2e tests via --label-filter
-hyperfleet-e2e test --label-filter=${LABEL_FILTER} --junit-report ${ARTIFACT_DIR}/junit.xml
+${E2E_BIN} test --label-filter=${LABEL_FILTER} --junit-report ${ARTIFACT_DIR}/junit.xml
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)